Changelog.md

```diff
@@ -7,6 +7,7 @@ Breaking Changes:
  * Command Line Interface: Remove the ``--old-reporter`` option.
  * Command Line Interface: Remove the legacy ``--ast-json`` option. Only the ``--ast-compact-json`` option is supported now.
  * General: Remove global functions ``log0``, ``log1``, ``log2``, ``log3`` and ``log4``.
+ * General: Enable ABI coder v2 by default.
```

Comment:
Not too big a deal, but actually: one line up?

Reply:
They get sorted before release. I actually often put them out of order myself, for example if I have multiple PRs that would otherwise conflict because of this.

Reply:
I know we'll sort them, as far as I'm concerned that should be the exception and definitely not happen intentionally... it's rather annoying if you want to git blame the entry, which I actually do quite often...

Reply:
I mean... it's also not the end of the world :-).